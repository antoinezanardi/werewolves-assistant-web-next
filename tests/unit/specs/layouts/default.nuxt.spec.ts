import type { mount } from "@vue/test-utils";

import Default from "~/layouts/default.vue";
import { mountSuspendedComponent } from "@tests/unit/utils/helpers/mount.helpers";

describe("Default Layout", () => {
  let wrapper: ReturnType<typeof mount<typeof Default>>;

  beforeEach(async() => {
    wrapper = await mountSuspendedComponent(Default);
  });

  it("should match snapshot when rendered.", () => {
    expect(wrapper).toBeTruthy();
    expect(wrapper.html()).toMatchSnapshot();
  });
});