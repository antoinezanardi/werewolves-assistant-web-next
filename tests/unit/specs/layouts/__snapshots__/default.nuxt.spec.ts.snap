// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Default Layout > should render component and match snapshot when mounted. 1`] = `
"<div id="default-layout" class="d-flex flex-column h-100">
  <nav-bar-stub></nav-bar-stub>
  <div class="flex-grow-1 h-100"></div>
</div>"
`;
